api/envoy/extensions/common/aws/v3/credential_provider.proto

```diff
@@ -14,18 +16,26 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: AWS common configuration]
 
-// Configuration for AWS credential provider. Normally, this is optional and the credentials are
+// Configuration for AWS credential provider. This is optional and the credentials are normally
 // retrieved from the environment or AWS configuration files by following the default credential
-// provider chain. This is to support cases where the credentials need to be explicitly provided
-// by the control plane.
+// provider chain. However, this configuration can be used to override the default behavior.
 message AwsCredentialProvider {
   // The option to use `AssumeRoleWithWebIdentity <https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithWebIdentity.html>`_.
-  // If inline_credential is set, this is ignored.
-  AssumeRoleWithWebIdentityCredentialProvider assume_role_with_web_identity = 1;
+  AssumeRoleWithWebIdentityCredentialProvider assume_role_with_web_identity_provider = 1;
```

Comment:
this will break the proto backward compatibility

Reply:
yep was hoping to sneak this in before the release :)